README.md

```diff
@@ -1,2 +1,184 @@
-# portfolio
- 
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Portfolio Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Portfolio Project <a name="about-project"></a>
+
+> A project for my Portfolio.
+
+**Portfolio Project** will showcase my skills and accomplishments as a developer.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The technologies used are HTML/CSS only.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML/CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Key features include:
+
+- **Mobile-First Design**
+- **Responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [N/A](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Steps to run: 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to clone and run this project you need:
+
+- A working browser
+- Installation of Git
+- Internet browser
+- Code editor like VS code
+- Live dev server plugin on VS code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install this project with: N/A
+
+### Usage
+
+To run the project, execute the following commands:
+
+- Run the index.html file using a live server
+- Make changes to the HTML/CSS
+- See the changes on your browser without refreshing
+
```

Comment:
- [ ] As the previous review suggested, I request that you clearly state the setup and installation procedures. This gives other developers the path to follow while installing and running your portfolio in their local machines 💻 .